README.md

```diff
@@ -13,7 +13,15 @@ qcloudapi-sdk-java是为了让Java开发者能够在自己的代码里更快捷
 1. 申请安全凭证。
 在第一次使用云API之前，用户首先需要在腾讯云网站上申请安全凭证，安全凭证包括 SecretId 和 SecretKey, SecretId 是用于标识 API 调用者的身份，SecretKey是用于加密签名字符串和服务器端验证签名字符串的密钥。SecretKey 必须严格保管，避免泄露。
 
-2. 下载SDK，放入到您的程序目录。
+2. 下载SDK，放入到您的程序目录。或使用Maven。
+3. Maven
+```
+<dependency>
+<groupId>com.qcloud</groupId>
+<artifactId>qcloud-java-sdk</artifactId>
+<version>2.0.6</version>
```

Comment:
Please provide a link, such as https://mvnrepository.com/artifact/com.qcloud/qcloud-java-sdk , for users to check all the versions. And add a note to indicate this is just an example for maven usage, 2.0.6 is not the final version.

Reply:
Thanks for your advice.